frontend/src/lib/components/metadata/MetricRange.svelte

```diff
@@ -1,5 +1,20 @@
 <script lang="ts">
 	import { metric, metricRange, metrics } from '$lib/stores';
+
+	function updateRange(e: Event, i: number) {
```

Comment:
Nit: Variable name for `i` is not super self-explanatory. If we change this and potentially add a function doctoring maybe we can make it easier to understand this function.